doc/source/news.rst

```diff
@@ -5,6 +5,653 @@
+       "グァグィグェグォ" \
+       WITH_CHECKS|WITH_TYPES
+
+     [
+       [
+         0,
+         0,
+         0
+       ],
+       {
+         "normalized":"ガギゲゴ",
+         "types":["katakana","katakana","katakana","katakana"],
+         "checks":[6,0,0,6,0,0,6,0,0,6,0,0],
+         "offsets":[0,6,12,18]
+       }
+     ]
+
+* [:doc:`reference/normalizers/normalizer_nfkc150`] Added a new option ``unify_katakana_di_sound``.
+
+  We can normalize "ヂ -> ジ" with this option.
+
+  .. code-block::
+
+     normalize \
+       'NormalizerNFKC130("unify_katakana_di_sound", true, \
+                          "report_source_offset", true)' \
+       "ヂ" \
+       WITH_CHECKS|WITH_TYPES
+     [
+       [
+         0,
+         0,
+         0
+       ],
+       {
+         "normalized":"ジ",
+         "types":["katakana"],
+         "checks":[3,0,0],
+         "offsets":[0]
+       }
+     ]
+
+* [:doc:`reference/normalizers/normalizer_nfkc150`] Added a new option ``unify_katakana_wo_sound``.
+
+  We can normalize "ヲ -> オ" with this option.
+
+  .. code-block::
+
+     normalize \
+       'NormalizerNFKC130("unify_katakana_wo_sound", true, \
+                          "report_source_offset", true)' \
+       "ヲ" \
+       WITH_CHECKS|WITH_TYPES
+     [
+       [
+         0,
+         0,
+         0
+       ],
+       {
+         "normalized":"オ",
+         "types":["katakana"],
+         "checks":[3,0,0],
+         "offsets":[0]
+       }
+     ]
+
+* [:doc:`reference/normalizers/normalizer_nfkc150`] Added a new option ``unify_katakana_z_sound``.
+
+  We can normalize "ズァ -> ザ", "ズィ -> ジ", "ズェ -> ゼ", and "ズォ -> ゾ" with this option.
+
+  .. code-block::
+
+     normalize \
+       'NormalizerNFKC130("unify_katakana_z_sounds", true, \
+                          "report_source_offset", true)' \
+       "ズァズィズェズォ" \
+       WITH_CHECKS|WITH_TYPES
+     [
+       [
+         0,
+         0,
+         0
+       ],
+       {
+         "normalized":"ズァズィズェズォ",
+         "types":["katakana","katakana","katakana","katakana","katakana","katakana","katakana","katakana"],
+         "checks":[3,0,0,3,0,0,3,0,0,3,0,0,3,0,0,3,0,0,3,0,0,3,0,0],
+         "offsets":[0,3,6,9,12,15,18,21]
+       }
+     ]
+
+* [:doc:`reference/normalizers/normalizer_nfkc150`] Added a new option ``unify_katakana_du_small_sound``.
+
+  We can "ヅァ -> ザ", "ヅィ -> ジ", "ヅェ -> ゼ", and "ヅォ -> ゾ" qith this option.
+
+  .. code-block::
+
+     normalize \
+       'NormalizerNFKC130("unify_katakana_du_small_sounds", true, \
+                          "report_source_offset", true)' \
+       "ヅァヅィヅェヅォ" \
+       WITH_CHECKS|WITH_TYPES
+     [
+       [
+         0,
+         0,
+         0
+       ],
+       {
+         "normalized":"ヅァヅィヅェヅォ",
+         "types":["katakana","katakana","katakana","katakana","katakana","katakana","katakana","katakana"],
+         "checks":[3,0,0,3,0,0,3,0,0,3,0,0,3,0,0,3,0,0,3,0,0,3,0,0],
+         "offsets":[0,3,6,9,12,15,18,21]
+       }
+     ]
+
```

Comment:
We also need to add a description about `unify_katakana_du_sound`.

Reply:
https://github.com/groonga/groonga/pull/1515

Reply:
And also `unify_katakana_trailing_o` https://github.com/groonga/groonga/pull/1506